README.md

```diff
@@ -1 +1,26 @@
-# Test-assignment2-team4
\ No newline at end of file
+# Test-assignment2-team4
+
+## https://meet.google.com/uhs-uydk-gyh?authuser=0
+
+## Problem Statement
+
+We will be creating a currency converter. The MVP is that the converter would convert pounds to dollars and dollars to pounds.
+
+## Roles
+
+| Names            | Job Roles     |
+| ---------------- | ------------- |
+| Safiya Abdulaahi | Administrator |
+| Keseana Robinson | QA            |
+| Augustine Osagie | Developer     |
+| James Spencer    | Dev Ops       |
+
+## Testing
+
+We will be running a smoke test as preliminary testing that comprises a set of non-exhaustive tests that verify that the build is stable enough for further testing.
+
+As we are building a currency converter, it is important to check that the mathmematical functions work
+
+## UI Design
+
+We will be using A/B testing to decide the UI will will be choosing for the currecny converter. James will be creating a google form where the team will pick out the better UI out of the two designs.
```

Comment:
In this section I will add the images from my UI design and the results from the Google Form